frontend/src/main.ts

```diff
@@ -35,6 +36,7 @@ function initializeApp(): void {
     })
   );
 
+  app.use(setupI18n());
```

Comment:
If there are no execution ordering dependencies, this might be worth introducing after Pinia and Router so that those things start loading sooner in the page load order.